test/e2e/cypress/integration/about.js

```diff
@@ -0,0 +1,20 @@
+context('Trento About page', () => {
+  beforeEach(() => {
+    cy.visit('http://demo.trento-project.io');
+  });
```

Comment:
we could make this an env variable with a default so we can use it in our local forks/envs

Reply:
You're actually right, I have to make a tiny library to make the import and the setup of this kind of stuff easier